<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formPrincipal</class>
 <widget class="QMainWindow" name="formPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modulo Principal</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;


border-color: rgb(4, 35, 38);</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>20</y>
      <width>532</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#037f8c;&quot;&gt;BEM VINDO AO  APOSTADORPRO&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#037f8c;&quot;&gt;Torne sua possibilidades em jogos certeiros&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sair">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_coletar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>275</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Coletar Dados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_visJogos">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>275</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Visualizar Jogos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_visPossibilidades">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>275</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Visualizar Possibilidades de Aposta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_simular">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>275</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Simular Aposta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
